packages/orbit-components/src/SkipLink/SkipLink.stories.tsx

```diff
@@ -37,231 +38,233 @@ const Content = (
 
-Playground.story = {
+export const Playground: Story = {
+  render: args => (
+    <>
+      <SkipLink {...args} />
+      {Content}
+    </>
+  ),
   parameters: {
     info: "All possible options for SkipLink. SkipLink is displayed only when focused. Use Tab or Shift + Tab to focus it.",
   },
-};
 
-Playground.args = {
-  links: [
-    {
-      href: "#guarantee",
-      name: "Go to kiwi guarantee information",
-    },
-    {
-      href: "#cancelation",
-      name: "Go to cancelation information",
-    },
-    {
-      name: "Reguest a refund",
-      onClick: action("Reguest a refund"),
-    },
-  ],
-  buttonLabel:
-    "Description for screen readers about where the user currently is or describing a more complex component.",
-  dataTest: "test",
-  id: "ID",
+  args: {
+    links: [
+      {
+        href: "#guarantee",
+        name: "Go to kiwi guarantee information",
+      },
+      {
+        href: "#cancelation",
+        name: "Go to cancelation information",
+      },
+      {
+        name: "Reguest a refund",
+        onClick: action("Reguest a refund"),
+      },
+    ],
+    buttonLabel:
+      "Description for screen readers about where the user currently is or describing a more complex component.",
+  },
 };
 
-export const WithinModal = () => (
-  <Modal onClose={action("onClose")} fixedFooter>
-    <SkipLink
-      links={[
-        {
-          name: "Go to outbound",
-          href: "#outbound",
-        },
-        {
-          name: "Go to inbound",
-          href: "#inbound",
-        },
-      ]}
-      buttonLabel="Description"
-    />
-    <ModalHeader
-      title="Enjoy something to eat while you fly"
-      illustration={<Illustration name="Meal" size="extraSmall" />}
-      description="Select a flight below to see the menu (where available)"
-    />
-    <ModalSection suppressed>
-      <Stack>
-        <Tooltip content={<div>Lorem ipsum dolor sit amet</div>}>
-          <Text uppercase weight="bold" id="outbound">
-            OUTBOUND
-          </Text>
-        </Tooltip>
-        <Card>
-          <CardSection
-            expandable
-            actions={
-              <Button type="secondary" size="small">
-                Edit
-              </Button>
-            }
-          >
-            <Stack direction="row" align="center" justify="between" spacing="200">
-              <CarrierLogo carriers={[{ code: "FR", name: "Ryanair" }]} size="large" />
-              <Stack spacing="100">
-                <Text size="small" type="secondary">
-                  Sat, Mar 31 Trip length: 1h55m
-                </Text>
-                <Stack direction="row" spacing="100" align="center">
-                  <Text weight="bold">London LHR</Text>
-                  <FlightDirect size="small" />
-                  <Text weight="bold">Prague PRG</Text>
+export const WithinModal: Story = {
+  render: () => (
+    <Modal onClose={action("onClose")} fixedFooter>
+      <SkipLink
+        links={[
```

Comment:
Because here the controls are disabled and it was like this 😅 But it's true that elsewhere where we are disabling controls, we have it in args anyway, moved.